corehq/apps/reports/templates/reports/partials/scheduled_reports_table.html

```diff
@@ -0,0 +1,135 @@
+{% load case_tags %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load report_tags %}
+{% load compress %}
+{% if scheduled_reports %}
+<table class="table table-striped table-bordered">
+    {% if not is_owner %}
+    <h3>Other Scheduled Reports (managed by others)</h3>
+    {% endif %}
```

Comment:
I would move this heading out to the main template, right above the line where the `others_scheduled_reports` table is included, for a couple reasons:
- An `<h3>` tag shouldn't be placed inside a `<table>`, so this should be moved up to before the `<table>` tag.
- This text isn't really part of the table, and is more a part of the page that includes the table; thus I think the main template is the better place to put it.